_posts/2019-12-09-reinvent-2019-recap.markdown

```diff
@@ -0,0 +1,130 @@
+---
+layout: post
+title:  "re:Invent 2019 Recap"
+date: 2019-12-09
+tags: aws reinvent recap
+author: Nathan Glover
+image: /img/blog/reinvent-2019-recap/reinvent-opener.png
+---
+
+## Introduction
+
```

Comment:
where is your Hero / Medallion shot?
Can you wrap up with a few thoughts about what next (for you and / or MR)?

Reply:
haha, I left the medallion shot out :) but i added closing thoughts